optaplanner-distribution-internal/src/main/assembly/scripts/runExamples.bat

```diff
@@ -2,9 +2,9 @@
 setLocal enableExtensions enableDelayedExpansion
 
 rem Most examples run (potentially slower) with max heap of 128 MB (so -Xmx128m), but 1 example's dataset requires 2 GB
-set "jvmOptions=-Xms256m -Xmx2g -Dorg.optaplanner.examples.dataDir=sources/data/"
+set "jvmOptions=-Xms256m -Xmx2g -Dorg.optaplanner.examples.dataDir=examples/sources/data/"
 set "mainClass=org.optaplanner.examples.app.OptaPlannerExamplesApp"
-set "mainClasspath=binaries/*;../binaries/*"
+set "mainClasspath=examples/binaries/*"
```

Comment:
The optaplanner binaries and example binaries have been merged into a single directory.